config/filter.d/sshd.conf

```diff
@@ -20,7 +20,7 @@ _daemon = sshd
 
 failregex = ^%(__prefix_line)s(?:error: PAM: )?[aA]uthentication (?:failure|error|failed) for .* from <HOST>( via \S+)?\s*$
             ^%(__prefix_line)s(?:error: PAM: )?User not known to the underlying authentication module for .* from <HOST>\s*$
-            ^%(__prefix_line)sFailed \S+ for .*? from <HOST>(?: port \d*)?(?: ssh\d*)?(: (ruser .*|(\S+ ID \S+ \(serial \d+\) CA )?\S+ %(__md5hex)s(, client user ".*", client host ".*")?))?\s*$
+            ^%(__prefix_line)sFailed \S+ for .*? from <HOST>(?: port \d*)?(?: ssh\d*)?(?:: | (?!from )|$)
```

Comment:
BTW we've already injecting test cases for the sshd-filter ... 
And the regexp, that suggested in this PR, is definitely injection-safe. Because otherwise this test case will fail (wrong 10.10.1.1 instead of correct 127.0.0.1).

See for example the test case "Injecting on username ssh ..." 2 lines above my newer test case ...
https://github.com/fail2ban/fail2ban/pull/1479/files#diff-affd3f5d7acb03d5fe1c1a6e773f05a6R118